javascript/reactjs-todo-davinci/playwright.config.ts

```
@@ -1,4 +1,6 @@
 import { defineConfig, devices } from '@playwright/test';
+import testConfig from './config.test.json';
+import testFidoConfig from './config.test.fido.json';
```

ESLint

[error] 'testFidoConfig' is defined but never used.

(no-unused-vars)